h2o-algos/src/main/java/hex/tree/DHistogram.java

```diff
@@ -195,7 +197,7 @@ public DHistogram(String name, final int nbins, int nbins_cats, byte isInt, doub
       _pred2 = cs._max;
       if (!cs.needsGammaDenom() && !cs.needsGammaNom()) {
         _vals_dim = Double.isNaN(_pred1) && Double.isNaN(_pred2) ? 3 : 5;
-        _dist = null; // intentionally cause NPE if used incorrectly
```

Comment:
please remove the commented code - not relevant anymore

Reply:
Fixed.